google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerImplTest.java

```diff
@@ -116,4 +117,20 @@ public void getDbclientAfterCloseThrows() {
       assertThat(e.getMessage()).contains("Cloud Spanner client has been closed");
     }
   }
+
+  @Test
+  public void exceptionIsTranslated() {
+    try {
+      SpannerImpl.runWithRetries(
+          new Callable<Object>() {
+            @Override
+            public Void call() throws Exception {
+              throw new Exception("Should be translated to SpannerException");
+            }
+          });
+    } catch (SpannerException e) {
+      assertThat(e.getErrorCode() == ErrorCode.INTERNAL);
```

Comment:
assertThat(e.getErrorCode()).isEqualTo(ErrorCode.UNKNOWN);
assertThat(e.getMessage()).contains("DML response missing stats possibly due to non-DML statement as input");
assertThat(e.getCause()).isInstanceOf(IllegalArgumentException.class);

Reply:
Fixed the first line .
I think the other two don't belong here in this test. Accidental paste perhaps?